multimodal/src/autogluon/multimodal/learners/few_shot_svm.py

```diff
@@ -379,21 +381,27 @@ def fit_per_run(
             svm=svm,
         )
 
-    def aggregate_column_features(self, features: Union[Dict, torch.Tensor]):
-        if isinstance(features, torch.Tensor):
+    def aggregate_column_features(
+        self,
+        features: Union[Dict, np.ndarray],
+        column_features_pooling_mode: Optional[str] = None,
+        is_train: Optional[bool] = True,
+    ):
+        if not is_train:
+            column_features_pooling_mode = self._config.data.column_features_pooling_mode
+
+        if isinstance(features, np.ndarray):
             return features
         elif isinstance(features, dict):
             assert len(features) != 0, f"column features are empty."
             if len(features) == 1:
                 return next(iter(features.values()))
-            if self._config.data.aggregate_column_features == "concat":
-                return torch.cat(features.values(), dim=1)
-            elif self._config.data.aggregate_column_features == "mean":
-                return torch.mean(torch.stack(features.values()), dim=0)
+            if column_features_pooling_mode == "concat":
+                return np.concatenate(list(features.values()), axis=1)
+            elif column_features_pooling_mode == "mean":
+                return np.mean(list(features.values()), axis=0)
```

Comment:
Wondering how features with different dimensions can be supported here (e.g. image feature versus text feature from different backbones)

Reply:
Currently, we only support single modality in few shot learning. The feature dimension should be consistent for different columns. The default is `concat` which is more flexible.